persistence/remote/src/test/resources/remote-cl-uri-config.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<infinispan
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="urn:infinispan:config:${infinispan.core.schema.version} https://infinispan.org/schemas/infinispan-config-${infinispan.core.schema.version}.xsd
+                          urn:infinispan:config:store:remote:${infinispan.core.schema.version} https://infinispan.org/schemas/infinispan-cachestore-remote-config-${infinispan.core.schema.version}.xsd"
+      xmlns="urn:infinispan:config:${infinispan.core.schema.version}"
+      xmlns:remote="urn:infinispan:config:store:remote:${infinispan.core.schema.version}" >
+
+   <!-- Default cache named to preserve old default cache name -->
+   <cache-container default-cache="RemoteStoreConfigUriTest">
+      <local-cache name="RemoteStoreConfigUriTest">
+         <persistence passivation="false">
```

Comment:
`passivation="false"` is the default behaviour, so this attribute can be removed.